modules/monitoring-assigning-tolerations-to-monitoring-components.adoc

```diff
@@ -30,14 +30,14 @@ metadata:
   namespace: openshift-monitoring
 data:
   config.yaml: |
-    *_component_*:
+    <component>:
       tolerations:
-        *_tolerations specification_*
+        <toleration specification>
```

Comment:
Per our style guide, we're replacing spaces with underscores: `<foo_variable>`

Reply:
Implemented everywhere in monitoring docs.